_sass/base/_markdown.scss

```diff
@@ -108,6 +113,12 @@ $article-line-height: 1.74;
     margin: 8px 0 24px;
   }
 
+  .date {
+    font-family: $main-font;
+    font-size: 14px;
+    color: rgba(black, .54);
```

Comment:
I found several usages of this `color: rgba(black, .54);` color and `color: rgba(black, .38);`. Maybe better to move them to variables. What do you think?

Reply:
Will be done with #327.